code/modules/admin/admin_verbs.dm

```diff
@@ -714,3 +717,18 @@ GLOBAL_LIST_INIT(admin_verbs_hideable, list(
 
 	log_admin("[key_name(usr)] has [AI_Interact ? "activated" : "deactivated"] Admin AI Interact")
 	message_admins("[key_name_admin(usr)] has [AI_Interact ? "activated" : "deactivated"] their AI interaction")
+
+
+/client/proc/give_faction_objective()//ST13 debugging, gives selected faction an objective
```

Comment:
This could probably be moved into its own file in the star trek folder

Reply:
Just one button, when we add more we will probably give it it's own admin tab/file